CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 2022.01.1
+
+**Released**: 2022.01.18
+
+**Commit Delta**: [Change from 2021.12.1 release](https://github.com/plus3it/spel/compare/2021.12.1...2022.01.1)
+
+**Manifests**: <https://github.com/plus3it/spel/blob/2022.01.1/manifests>
+
+**Summary**:
+
+*   Publishes image for CentOS 8 Stream
+*   Removes image CentOS 8
```

Comment:
I think we're still keeping the previously published old CenOS 8 AMIs.  Just not publishing new CentOS 8.

Reply:
Yeah, let's say something like:
```suggestion
*   Deprecates CentOS 8 image -- published `minimal-centos-8-hvm` images will remain available for the time being
```